lib/model/__init__.py

```diff
@@ -7,7 +7,7 @@ def __init__(self, classifier, parameters={}, gs_params={}):
         self.classifier = classifier
         self.model = classifier(**parameters)
         self.parameters = parameters
-        self.gs_model = GridSearchCV(self.model, gs_params)
+        self.gs_model = GridSearchCV(self.classifier(), gs_params)
```

Comment:
`self.model` is used to evaluate training and prediction of the vanilla classifier, so we use it later in the process.